Mathlib/LinearAlgebra/RootSystem/Hom.lean

```diff
@@ -651,6 +651,12 @@ instance : DistribMulAction P.Aut M where
   smul_zero w := show weightHom P w 0 = 0 by simp
   smul_add w x y := show weightHom P w (x + y) = weightHom P w x + weightHom P w y by simp
 
+@[simp] lemma reflection_smul (i : ι) (x : M) : Equiv.reflection P i • x = P.reflection i x := rfl
+
+lemma smul_root (i : ι) (g : P.Aut) :
+    g • P.root i = P.root (g.indexEquiv i) := by
```

Comment:
Would it make sense to provide the other direction as a `simp`-lemma named `root_indexEquiv`?

Reply:
Good point, done!